ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1336,6 +1333,11 @@ func (r *RayClusterReconciler) calculateStatus(ctx context.Context, instance *ra
 					Reason:  rayv1.AllPodRunningAndReadyFirstTime,
 					Message: "All Ray Pods are ready for the first time",
 				})
+
+				// Record ray_cluster_provisioned_duration_seconds duration metric
```

Comment:
The metric should not be recorded in `calculateStatus`. It should only be recorded when the status update succeeds, to avoid counting it more than once.